### DISCUSSION TITLE
New Tabs

### DISCUSSION THREAD

**Discussion by gabriel20xx** [sdwebui-w-horde/sd-webui-stable-horde-worker#126] [Ideas]
I'm currently working on this plugin and I wanna split the plugin in the following tabs:
Generation, User, Worker, Kudos, Team, News, Status, Stats and Settings

My question is now, what do you wanna see on the User, Worker and Teams tab?

**Comment by MaikoTan** [comment]
Sounds good! :+1:

For the User and Team tab, maybe not only showing the information of current user but also others?
As for the Worker, should still have feature for edit the basic information of owned worker and also switches for maintenance mode?
